paddlenlp/transformers/electra/modeling.py

```diff
@@ -199,7 +199,17 @@ def __init__(self, vocab_size, embedding_size, hidden_dropout_prob,
         self.layer_norm = nn.LayerNorm(embedding_size, epsilon=layer_norm_eps)
         self.dropout = nn.Dropout(hidden_dropout_prob)
 
-    def forward(self, input_ids, token_type_ids=None, position_ids=None):
+    def forward(self,
+                input_ids,
+                token_type_ids=None,
+                position_ids=None,
+                inputs_embeds=None):
+
+        if input_ids is not None:
+            input_embeddings = self.word_embeddings(input_ids)
+        else:
+            input_embeddings = inputs_embeds
```

Comment:
I think this code block can be improved to:

```python
        if input_ids is None:
            inputs_embeds = self.word_embeddings(input_ids)
```

and in the `forward` method, rename `input_embeddings` to `inputs_embeds`.  In this way, the code looks more concise. how do you think about it?

Reply:
Should the `input_embeddings = self.word_embeddings(input_ids)` in following original code be removed

```
        if token_type_ids is None:
            token_type_ids = paddle.zeros_like(input_ids, dtype="int64")
        input_embeddings = self.word_embeddings(input_ids)
        position_embeddings = self.position_embeddings(position_ids)
        token_type_embeddings = self.token_type_embeddings(token_type_ids)
        embeddings = input_embeddings + position_embeddings + token_type_embeddings
```

Reply:
Thanks for the review folks.
@wj-Mcat I agree that renaming `input_embeddings` to `inputs_embeds` makes the code more concise, it also makes it less explicit/readable.
@guoshengCS good call. Removed the redundant line of code.